src/client-services/github.client-service.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosResponse, AxiosInstance } from 'axios';
```

Comment:
I always find return types ugly.

: AxiosInstance

and definition types like

: string | undefined

Reply:
@SamHuo213  Thanks for your comment. Could you please make it clearer? What exactly is in your mind about the return type?

Reply:
I prefer not having return types. Since we don't have explicit standards yet, I am happy to move forward for now.

e.g.

function axiosConfig() {
    ....
}